src/components/ZclDomainClusterView.vue

```diff
@@ -232,6 +232,24 @@ export default {
   props: ['domainName', 'clusters'],
   mixins: [CommonMixin],
   computed: {
+    clusterSelectionOptions() {
+      if (
+        this.$store.state.zap.selectedZapConfig?.zclProperties?.category ===
+        'matter'
```

Comment:
Ditto

Reply:
yeah, I am about to push a fix to centralize this but as discussed:
1.) there is no avoiding conditionals
2.) zcl.json changes would be much harder for a ZAP developer to make 
3.) we already implement this from a .zap file so if we want to do a redesign then it would have to be a new ticket